metabase/metabase
# Can't choose recently viewed link with the mouse when editing dashboard link card #35037

Closes https://github.com/metabase/metabase/issues/35037

### Description

Ensures that clicking on a search/recents entry only triggers the onClick action, rather than the href/redirect behavior that the issue brings up.

_What Changed:_
- Added a new prop `isEnabled` to the InfoTextTableLink component.
- Modified the href attribute of a ResultTitle component.


### How to verify
 
- Create a new dashboard A, save it and visit it
- Create a new dashboard B and edit it.
- Add a link card to it
- A dropdown with "Recently viewed" items will appear
- Click header of the first entry in it with a mouse


### Checklist
